Helper_Scripts/TTS_Installers/install_tts_chatterbox.py

```
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)